src/adr/0003-distributed-workqueue.md

```diff
@@ -6,6 +6,8 @@ Date: 2019-10-02
 
 Accepted (lazy consensus)
 
+Superceded by [16. Distributed WorkQueue](0016-distributed-workqueue.md)
```

Comment:
Ah, very good case: do we keep old stuff in the ADR or do we drop them as soon as they don't describe the current state of things?

Reply:
I have understood that the principle is to keep old stuff.
And it's natively handled by the tool, here I did `adr new -s 3 Distributed WorkQueue` to generate the new ADR and have automatically the `Superceded` comment added to the previous one. So it seems to be the way to go.

Reply:
great, thank you for the explanation, let's use the same rules as other people.